test/format-support.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-env mocha */
+'use strict'
+
+const chai = require('chai')
+const dirtyChai = require('dirty-chai')
+const expect = chai.expect
+chai.use(dirtyChai)
+const BlockService = require('ipfs-block-service')
+const dagCBOR = require('ipld-dag-cbor')
+
+const IPLDResolver = require('../src')
+
+module.exports = (repo) => {
+  describe('IPLD format support', () => {
+    let data, cid
+
+    before((done) => {
+      const bs = new BlockService(repo)
+      const resolver = new IPLDResolver({ blockService: bs })
+
+      data = { now: Date.now() }
```

Comment:
Why the `Date.now()`? I generally prefer tests that do the exact same across runs.